ansible/roles/logstash/templates/pipeline.conf

```diff
@@ -0,0 +1,39 @@
+input {
+    beats { 
+        port => 5044
+    }
+    file {
+        path= "/media/ephemeral0/logs/accumulo/*.log"
+    }
+}
+filter {
+  if [fileset][module] == "logstash" {   
+      if [fileset][name] == "hadoop" {
+    grok {
+        match => { "message" => "%{TIMESTAMP_ISO8601}%{SPACE}%{WORD:log level}%{GREEDYDATA:hostname}%{GREEDYDATA:LINK}:%{GREEDYDATA:state}%{WORD:module}: %{GREEDYDATA:message}"}
```

Comment:
Please add an example of a like being matched before each match statement.  That will make this easier to read and validate.